models/intermediate/int__daily_shopify_customer_orders.sql

```diff
@@ -15,7 +15,10 @@ with orders as (
         source_relation,
         count(distinct product_id) as count_products,
         count(distinct product_id || '-' || variant_id) as count_product_variants,
-        sum(quantity) as sum_quantity
+        sum(quantity) as sum_quantity,
+        sum(price) as sum_price,
+        sum(pre_tax_price) as sum_pre_tax_price,
+        sum(total_discount) as sum_total_discount
```

Comment:
- Not directly related to the dependency updates, but I noticed a dollar-based field was never part of this orders model, and I think naturally an orders model should provide some dollar field in addition to product/quantity count. Then people can look at total a customer spends, which I'm sure would be a frequent use case for any aggregate customer/order model. Let me know if you a specific reason not to include this?

Reply:
deleting this-- there are more dollar based fields brought in from shopify